### ISSUE TITLE
Unable to disable collaborative editing

### ISSUE THREAD

**Issue by daladim** [outline/outline#3388]
Thanks for this great piece of software!

On my self-hosted test instance, collaborative mode is enabled (by default). I'd like to disable it but the switch to control this behaviour is disabled (I cannot click it).

Reading at the code (but I'm not a web-development expert), there is
```
        <Switch
          id="collaborativeEditing"
          name="collaborativeEditing"
          checked={data.collaborativeEditing}
          disabled={data.collaborativeEditing}
          onChange={handleChange}
        />
```
which looks like this switch is disabled if (and only if) collaborative editing is enabled. Is this a bug, or is it by design that an instance that has opted in collaborative editing cannot go back?

Thanks

## Versions

I am using Outline 0.63.0-1, and the latest redis, postgres and minio dependencies, using the recommended Docker Compose setup

## Context

In case you're interested :)
I'm trying Outline (and am puzzled by the recent changes regarding collaborative editing).
I'd like to revert to non-collaborative editing, because (and I would definitely love at least #3132) I'm not adverse to collaborative editing, but I am to this "always-edit" mode, that feels like it is easy to inadvertently delete/change some content (yes, the history is kept so that's not a big deal...unless we inadvertently change something _without noticing_)

**Comment by daladim** [comment]
As a workaround (or for testing reasons), is there a way (environment variable, Docker settings...) to disable collaborative editiing on 0.63.0-1? (ideally without changing anything in the Docker container)

**Comment by daladim** [comment]
I could run Outline with this setting disabled by migrating from a 0.62.0 version with collaborative editing disabled...but I'm sure there should be an easier way :)

**Comment by tommoor** [comment]
This is by design as disabling collaborative editing after enabling will result in data loss, non-collab editing will also be removed in a future version of the software so that there aren't two different code paths to maintain.

If you've just installed and are not worried about data loss you can either disable in the database directly or through the API as two workaround options 😄

**Comment by daladim** [comment]
All right, thanks for your answer.
I may try this on this weekend then.

(and Happy Easter)

**Comment by daladim** [comment]
Just to be sure, what "data loss" are we talking about? Will we lose data of documents being currenlty edited? (and as such, it is "safe" to disable this setting via the DB on an instance that is not in use) Or could we lose already edited documents already edited?

**Comment by tommoor** [comment]
When collaborative editing is enabled the data is stored in a different database column. So if you:

- disable it
- edit documents
- re-enable it

Documents will revert to the state they were in when you first disabled it, losing any intermediate changes

**Comment by Redmauss** [comment]
If anyone wants to know, here is the command I used to disable collaborative editing in psql:
`UPDATE teams SET "collaborativeEditing" = 'f' WHERE 1=1;`

Also, I love outline.  I use it daily.  But let me say I have a _very_ strong disapproval of a checkbox that creates a permanent change with zero warning.  I just spent the last two hours trying to figure out why I couldn't change the slider, only to find out that I was never intended to.  There should be a confirmation or something if it's going to permanently enable.  Just my two cents.  
Also one more thing, my CPU usage skyrocketed when I had collaborative editing enabled.  Part of the reason I was so desperate to change it back.  Not sure what that's about.
